packages/@aws-cdk/cloud-assembly-schema/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@aws-cdk/cloud-assembly-schema",
+  "version": "0.0.0",
+  "description": "Cloud Assembly Schema",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "jsii": {
+    "outdir": "dist",
+    "targets": {
+      "java": {
+        "package": "software.amazon.awscdk.cloudassembly.schema",
```

Comment:
I think this should be `.awscdk.cloudassemblyschema` no?

Reply:
Thats was my thought as well, but I followed the same naming as [cdk-assets-schema](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/cdk-assets-schema/package.json#L11).